docs/docsite/rst/playbook_guide/playbooks_variables.rst

```diff
@@ -101,20 +101,42 @@ Boolean variables
 =================
 
 Ansible accepts a broad range of values for boolean variables: ``true/false``, ``1/0``, ``yes/no``, ``True/False`` and so on. The matching of valid strings is case insensitive.
-While documentation examples focus on ``true/false`` to be compatible with ``ansible-lint`` default settings, you can use any of the following:
+While documentation examples focus on ``true/false`` to be compatible with ``ansible-lint`` default settings. You can use any of the following values:
+
+Native Boolean
```

Comment:
I would avoid using "native boolean" for anything else than `true`/`false` because the only values that are native booleans in all versions of YAML specification.

Maybe we should mention that bit. Currently Ansible uses a YAML 1.1 loader, but when it will move to a 1.2 loader, only these values will remain booleans. This might help people understand better the long term implications.

Reply:
the issue is not only YAML, Ansible has internal functions to convert booleans that follow the YAML 1.1 spec, so this can apply to JSON and Jinja2.

Reply:
> I would avoid using "native boolean" for anything else than true/false because the only values that are native booleans in all versions of YAML specification.

I can change header to `Recognizable as Boolean` (or smth like that) for that section,  to separate it from `Native boolean` definition in yaml 1.2

And as I see, that section of `Recognizable as Boolean\Native boolean` still has to to be present here for new ansible version with new loader if it will support Core Schema of[ yml 1.2](https://yaml.org/spec/1.2.2/#103-core-schema) because there still several valid variants of inputs `true | True | TRUE | false | False | FALSE`.
If not (It will wotk only with [Json Schema](https://yaml.org/spec/1.2.2/#102-json-schema)) - in new version ofk make sense to reduce it to only two values.

PS: We can add some note like:  just use  `true\false` because that easier for everyone around

Reply:
@ssbarnea 
Could u plz clarify about yaml schema and name of segment to change PR

Reply:
@skilyazhnev Thanks for keeping at this one.

How about we keep the "Native boolean" heading but put only `true` / `false` underneath that to reflect what @ssbarnea has mentioned about the spec: https://yaml.org/spec/1.2-old/spec.html#id2803629

For the other values in the current table, maybe we could rename the heading to "Implicit boolean"? Something like this:

```
Native boolean
--------------

Ansible treats these values as native booleans:

<table goes here>

Implicit boolean
------------------------

Ansible recognizes these values as booleans:

<table goes here>

```